tests/integration/ethers-cli/setup_node_v20.sh

```diff
@@ -3,14 +3,14 @@
 # Discussion, issues and change requests at:
 #   https://github.com/nodesource/distributions
 #
-# Script to install the NodeSource Node.js 18.x repo onto a
+# Script to install the NodeSource Node.js 19.x repo onto a
```

Comment:
Should these be `20.x`?

Reply:
It's the script that pulled from `https://deb.nodesource.com/setup_20.x`, but it does look like they forgot to update that part

Reply:
updated for make better